src/backend/Texture.h

```diff
@@ -44,6 +44,7 @@ namespace backend {
             void UpdateUsageInternal(nxt::TextureUsageBit usage);
 
             static bool IsDepthFormat(nxt::TextureFormat format);
+            static bool IsStencilFormat(nxt::TextureFormat format);
```

Comment:
nit: Can you move both of these to be outside of the TextureBase class and next to TextureFormatPixelSize?